components/eam/src/chemistry/mozart/mo_gas_phase_chemdr.F90

```diff
@@ -1311,7 +1313,7 @@ subroutine gas_phase_chemdr(lchnk, ncol, imozart, q, &
     call t_startf('aero_model_gasaerexch')
     call aero_model_gasaerexch( imozart-1, ncol, lchnk, delt, latndx, lonndx, reaction_rates, &
                                 tfld, pmid, pdel, mbar, relhum, &
-                                zm,  qh2o, cwat, cldfr, ncldwtr, &
+                                zm,  qh2o, cwat_liq, cldfr, ncldwtr, &
```

Comment:
Should the cloud fraction (cldfr) here be total or just liquid cloud fraction?

Reply:
Based on what I checked on the aqueous chemistry code, cldfr is only used for converting the grid values to be in-cloud values (in mo_setsox.F90), so the total cloud fraction should be fine.

Reply:
If the cloud fraction is used to convert grid-mean liquid cloud water to cloud-mean value, liquid cloud fraction should be used instead of total cloud fraction. Otherwise, this could give a potentially strong bias in ice-dominant grids.

Reply:
It is not used for converting grid-mean liquid cloud water to cloud-mean value. It is only used for converting the grid value of formed sulfate from the aqueous chemistry to be in-cloud value.